edx_exams/apps/api/v1/views.py

```diff
@@ -56,13 +56,16 @@ class CourseExamsView(ExamsAPIView):
     """
 
     authentication_classes = (JwtAuthentication,)
+
+    # Staff = instructor in this case, can access all users
```

Comment:
NIT: not sure if this comment should be permanent

Reply:
I didn't make that comment, but I can delete it if you'd like! Also thanks for the clarification Zach.

Reply:
I have now rebased off main!